CLI/README.md

```diff
@@ -0,0 +1,54 @@
+![logo](./docs/logo-cli.png)
+
+# axe DevTools CLI Example
+
+This example project demonstrates how axe DevTools CLI spec mode is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
+
+## Prerequisites
+
+- Install the [Driver](https://docs.deque.com/devtools-html/4.0.0/en/cli-install-home#requirements) for the browser you choose to use for the testing.
+- You need a valid license to use our CLI. For more information, see [Install from Deque’s Agora](https://docs.deque.com/devtools-html/4.0.0/en/cli-install-nodejs) page.
+
+## Clone Project
+
+Follow these steps to clone and navigate to the directory:
+1. Clone this repo from GitHub.
+2. Open the command window and navigate from the root of the repo to this example with the following command:
+
+```sh
+cd CLI
+```
+
+You can use CLI in two modes:
+- Spec Mode
+- URI Mode
+
+This example demonstrates spec mode. For URI mode, see our [axe DevTools HTML documentation](https://docs.deque.com/devtools-html/4.0.0/en/cli-quickstart-pages)
```

Comment:
```suggestion
This example demonstrates spec mode. For more information on URI mode, see our [documentation](https://docs.deque.com/devtools-html/4.0.0/en/cli-quickstart-pages)
```

Reply:
> 

We didn't furnish any information on URI mode. So, I directly referred to the Documentation.

Reply:
https://docs.deque.com/devtools-html/4.0.0/en/cli-page-tests -> this is the main page documenting URI mode in CLI.

Reply:
> https://docs.deque.com/devtools-html/4.0.0/en/cli-page-tests

updated cross-reference